### DISCUSSION TITLE
UCB Data100: Principles and Techniques of Data Science - csdiy.wiki

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#559] [Announcements]
# UCB Data100: Principles and Techniques of Data Science - csdiy.wiki

CS自学指南

https://csdiy.wiki/en/%E6%95%B0%E6%8D%AE%E7%A7%91%E5%AD%A6/Data100/

**Comment by vietnguyenba12** [comment]
How can i access to the assignment and lab of this course?. It seemed only for UCB's students

**Reply by fangtaoysu** [reply]
You can learn early course, they are opening.😉
